doc/build_apps/run_app.rst

```diff
@@ -1,20 +1,35 @@
 Running CCF Applications
```

Comment:
Should this be more specific? e.g. "Running CCF Applications in Sandbox"? Otherwise it suggests that this is a generic way to deploy CCF applications.

Reply:
I've tried to clarify that with a comment below:

> ``sandbox.sh`` is a demonstration tool, and not intended for use in production deployments.